### ISSUE TITLE
DistanceBand include the point itself as neighbor

### ISSUE THREAD

**Issue by qcts33** [pysal/pysal#779]
The behavior is not the same as the example. The DistanceBand include the point itself as neighbor.

```python
points=[(10, 10), (20, 10), (40, 10), (15, 20), (30, 20), (30, 30)]

w=pysal.DistanceBand(points,threshold=11.2,binary=False)
C:\Miniconda3\lib\site-packages\pysal\weights\Distance.py:499: RuntimeWarning: divide by zero encountered in double_scalars
  weights[i].append(weight**self.alpha)

w.weights
Out[29]: 
{0: [0.10000000000000001, inf, 0.089442719099991588],
 1: [0.10000000000000001, 0.089442719099991588, inf],
 2: [inf],
 3: [inf, 0.089442719099991588, 0.089442719099991588],
 4: [0.10000000000000001, inf],
 5: [0.10000000000000001, inf]}

w.neighbors
Out[30]: {0: [1, 0, 3], 1: [0, 3, 1], 2: [2], 3: [3, 0, 1], 4: [5, 4], 5: [4, 5]}
```

I'm using anaconda with python 3.5.1, scipy 0.17.0, numpy 1.10.4, pysal 1.11.0

**Comment by sjsrey** [comment]
This is related to a bug #726 that has been fixed in [dev](https://github.com/pysal/pysal/tree/dev). It should be out in a rolling release in the next few weeks, but for now one could grab the dev branch as an interim solution.

**Comment by sjsrey** [comment]
Fixed in [1.11.1](https://github.com/pysal/pysal/releases/tag/v1.11.1)
